space-plugins/nuxt-starter/app.config.ts

```diff
@@ -1,14 +1,8 @@
 export default defineAppConfig({
+	type: 'space-plugin',
 	appBridge: {
-		type: 'space-plugin',
 		enabled: true,
 		oauth: true,
 		origin: 'https://app.storyblok.com',
 	},
-	auth: {
-		endpointPrefix: '/api/connect',
-		initOauthFlowUrl: `/api/connect/storyblok`,
-		successCallback: '/',
-		errorCallback: '/401',
-	},
 });
```

Comment:
Now we no longer need to have full appConfig. Only things that we want to override.